core/sql/regress/udr/EXPECTED100.SB

```diff
@@ -1706,7 +1706,7 @@ OUT2
 >>
 >>obey TEST100(udrtrace);
 >>log;
-Trafodion Conversational Interface @version@
+Conversational Interface @version@
```

Comment:
Leaving the old filters as is for now since I noticed they are spread across other filter files.The new filter is more general and should cover future copyright changes as well.